benchmarks_runner/eval.py

```diff
@@ -81,29 +82,31 @@ def evaluate_results(
                     message = response.get('message', None)
                     if message is None:
                         # most likely came from the ARS
-                        message = ((response.get('fields') or {}).get('data') or {}).get('message', {})
+                        message = reduce(lambda val, key: val.get(key) if val else {}, ["fields", "data", "message"], response)
         elif type(results_dir) is dict:
             if uid not in results_dir:
                 warnings.warn(f'Results for query {uid} were not found.')
             else:
                 # Grab message from dict
-                message = results_dir.get(uid, {}).get("message", {})
+                message = reduce(lambda val, key: val.get(key) if val else {}, [uid, "message"], results_dir)
```

Comment:
Same situation as above
```suggestion
                message = reduce(lambda val, key: {} if val.get(key) is None else val[key], [uid, "message"], results_dir)
```